iceoryx_posh/test/moduletests/test_roudi_memory_manager.cpp

```diff
@@ -70,9 +92,17 @@ TEST_F(RouDiMemoryManager_Test, CallingCreateMemoryWithMemoryProviderSucceeds)
     sut.addMemoryProvider(&memoryProvider2);
 
     EXPECT_THAT(sut.createAndAnnounceMemory().has_error(), Eq(false));
+}
 
-    EXPECT_CALL(memoryBlock1, destroyMock());
-    EXPECT_CALL(memoryBlock2, destroyMock());
```

Comment:
I would not remove this. How do you check that the memory destruction was triggered?